go/runtime/bundle/bundle.go

```diff
@@ -277,6 +280,9 @@ func (bnd *Bundle) Write(fn string) error {
 		return fmt.Errorf("runtime/bundle: failed to write bundle: %w", err)
 	}
 
+	// Update the manifest hash.
+	bnd.manifestHash = bnd.Manifest.Hash()
```

Comment:
Feels like this does not belong here. Should this be moved to `Add`, where we update the manifest?

Reply:
Hm. My thinking is that `manifestHash` should equal the hash of the manifest that is _written on disk_. Until `Write` is called nothing is written on disk so the hash should not be updated.

Reply:
Doesn't this contradicts the comment?
> manifestHash is the original manifest hash of the bundle at time the bundle was loaded.

I would expect that manifest hash is consistent with manifest in the bundle (in memory).

Reply:
No, the entire point is to _not_ be what is in memory as that can be modified (and you can always get the in-memory hash using `bnd.Manifest.Hash()`). The only place where this private `manifestHash` is used is the computation of exploded bundle paths.